Assets/GothicVR/Scripts/Npc/Actions/AnimationActions/UseItemToState.cs

```diff
@@ -1,55 +1,88 @@
 using GVR.Caches;
 using GVR.Creator;
+using GVR.GothicVR.Scripts.Manager;
 using UnityEngine;
+using ZenKit.Daedalus;
 
 namespace GVR.Npc.Actions.AnimationActions
 {
     public class UseItemToState : AbstractAnimationAction
     {
         
-        private const string animationStartScheme = "T_{0}_STAND_2_S{1}" ;
-        private const string animationEndScheme = "T_{0}_S{1}_2_STAND" ;
-        
+        private const string animationScheme = "T_{0}_{1}_2_{2}";
+        private const string LoopAnimationScheme = "S_{0}_S{1}";
+
+        private int itemToUse => Action.Int0;
+        private int desiredState => Action.Int1;
+
+
         public UseItemToState(AnimationAction action, GameObject npcGo) : base(action, npcGo)
         { }
 
         public override void Start()
         {
-            // Nothing in hand && new item shall be put into hand
-            if (Props.itemAnimationState < 0 && Action.Int1 >= 0)
+            PhysicsHelper.DisablePhysicsForNpc(Props);
```

Comment:
Why is physics deactivated here? is that a hack / workaround or is there a proper reason for it?

Reply:
Good point. I added a comment:
```c#
            // We need to disable physics as e.g. Gomez won't sit on his throne otherwise. (Animations are aligned with the objects they use already).
            // A collider would break it.
```